daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -466,6 +467,29 @@ cleanTests baseDir = testGroup "daml clean"
                                 , unlines (map ("       "++) filesAtEnd)
                                 ]
 
+templateTests :: TestTree
+templateTests = testGroup "templates"
+    [ testCase name $ do
+          withTempDir $ \dir -> withCurrentDirectory dir $ do
+              callCommandQuiet $ unwords ["daml", "new", "foobar", name]
```

Comment:
This should be fine as long as we never add a space in a template name.

Reply:
Spaces in template names seems like a pretty terrible UX so I’m happy to have that blow up in tests :slightly_smiling_face: